ST_environs.c

```diff
@@ -233,14 +178,12 @@ static void _set_temp_reduction( void) {
   }
 
 #ifdef STEPWAT
```

Comment:
I believe this #ifdef statement can be removed as well?

Reply:
Yep! Commit 51d6f29 resolves this.